package/MDAnalysis/topology/LAMMPSParser.py

```diff
@@ -318,23 +414,29 @@ def read_DATA_timestep(self, n_atoms, TS_class, TS_kwargs,
         .. versionchanged:: 0.18.0
            Added atom_style kwarg
         """
-        if atom_style is None:
-            self.style_dict = None
-        else:
+        # handling invalid atom_style keywords:
+        if atom_style in ATOM_STYLES.keys():
```

Comment:
This is repeated so can be a helper function.

Reply:
I defined a new method *invalid_atom_style*.